src/app/space/create/deployments/apps/collapsible-deployment-info.component.spec.ts

```diff
@@ -0,0 +1,160 @@
+import { Component, DebugElement, Input } from '@angular/core';
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { By } from '@angular/platform-browser';
+
+import { Observable } from 'rxjs';
+
+import { times } from 'lodash';
+
+import { CollapseModule } from 'ngx-bootstrap/collapse';
+
+import { createMock } from 'testing/mock';
+
+import { Environment } from '../models/environment';
+import { DeploymentsService } from '../services/deployments.service';
+
+import { CollapsibleDeploymentInfoComponent } from './collapsible-deployment-info.component';
+
+// Makes patternfly charts available
+import { ChartModule } from 'patternfly-ng';
+import 'patternfly/dist/js/patternfly-settings.js';
+
+@Component({
+  selector: 'deployments-donut',
+  template: ''
+})
+class FakeDeploymentsDonutComponent {
+  @Input() mini: boolean;
+  @Input() spaceId: string;
+  @Input() applicationId: string;
+  @Input() environment: Environment;
+}
+
+@Component({
+  selector: 'deployment-graph-label',
+  template: ''
+})
+class FakeDeploymentGraphLabelComponent {
+  @Input() type: any;
+  @Input() dataMeasure: any;
+  @Input() value: any;
+  @Input() valueUpperBound: any;
+}
+
+@Component({
+  selector: 'pfng-chart-sparkline',
+  template: ''
+})
+class FakePfngChartSparkline {
+  @Input() config: any;
+  @Input() chartData: any;
+}
+
+describe('CollapsibleDeploymentInfoComponent', () => {
+  let component: CollapsibleDeploymentInfoComponent;
+  let fixture: ComponentFixture<CollapsibleDeploymentInfoComponent>;
+  let mockSvc: jasmine.SpyObj<DeploymentsService>;
+  let mockEnvironment: Environment;
+
+  beforeEach(() => {
+    mockSvc = createMock(DeploymentsService);
+    mockSvc.getVersion.and.returnValue(Observable.of('1.2.3'));
+    mockSvc.getCpuStat.and.returnValue(Observable.of({ used: 1, quota: 2 }));
+    mockSvc.getMemoryStat.and.returnValue(Observable.of({ used: 3, quota: 4, units: 'GB' }));
+    mockSvc.getAppUrl.and.returnValue(Observable.of('mockAppUrl'));
+    mockSvc.getConsoleUrl.and.returnValue(Observable.of('mockConsoleUrl'));
+    mockSvc.getLogsUrl.and.returnValue(Observable.of('mockLogsUrl'));
+    mockSvc.deleteApplication.and.returnValue(Observable.of('mockDeletedMessage'));
+
+    mockEnvironment = { name: 'mockEnvironment' } as Environment;
+
+    TestBed.configureTestingModule({
+      imports: [CollapseModule.forRoot()],
+      declarations: [
+        CollapsibleDeploymentInfoComponent,
+        FakeDeploymentsDonutComponent,
+        FakeDeploymentGraphLabelComponent,
+        FakePfngChartSparkline
+      ],
+      providers: [
+        { provide: DeploymentsService, useValue: mockSvc }
+      ]
+    });
```

Comment:
Can you use the TestContext util here? There are a bunch of examples of this throughout the deployments area codebase now. `resource-card.component.spec.ts` looks like a good one to reference.